src/services/findAllReferences.ts

```diff
@@ -2014,16 +2030,40 @@ namespace ts.FindAllReferences {
             readonly symbol: Symbol;
             readonly kind: NodeEntryKind | undefined;
         }
+
+        function isStatic(symbol: Symbol): boolean{
+            if(!symbol?.valueDeclaration) {return false;}
+            if(symbol.valueDeclaration?.modifierFlagsCache) {return !!((symbol.valueDeclaration.modifierFlagsCache & ModifierFlags.Static));}

```

Comment:
Can you update your changes to reflect the style of the rest of the file (space after keywords etc.)? The editor's default formatting works well here.

Reply:
fixed.